Affiliated-project-FAQ.md

```diff
@@ -1,6 +1,6 @@
 # Open Bioinformatics Foundation affiliated and core project FAQ
 
-## Q: How do I apply for OBF Affiliated Project status?
+## Q: How do I apply to become an OBF Affiliated Project?
```

Comment:
Don't we need to say Affiliated _or_ Candidate OBF Project if it is needed for both? Or did I misunderstand what you meant?
```suggestion
## Q: How do I apply to become an Affiliated or Candidate OBF Project?
```

Reply:
I think my original assumption was that you automatically move through the stages once you apply for Candidate status, so it didn't need to be explicit, but either is fine.